packages/samples/headless-ssr-commerce/app/search/page.tsx

```diff
@@ -22,8 +22,19 @@ export default async function Search() {
 
   // Fetches the static state of the app with initial state (when applicable)
   const staticState = await searchEngineDefinition.fetchStaticState({
-    controllers: {cart: {initialState: {items}}},
+    controllers: {
+      cart: {initialState: {items}},
+      context: {
+        language: 'en',
+        country: 'US',
+        currency: 'USD',
+        view: {
+          url: '?',
+        },
+      },
```

Comment:
This is awkward to have to define the `view.url` here in a search page. Ideally we would make it so when fetchStaticState is called from a searchEngineDefinition not every type is required. I am not sure how to achieve this tho. We could also just make the `view` type optional.

Reply:
Talked with Oli and he said it is fine. We should set the url everywhere. CAPI could  theoretically use the url defined in search somewhere down the line.